code_annotations/helpers.py

```diff
@@ -113,25 +114,59 @@ def get_annotation_regex(annotation_regexes):
     """
     Return the full regex to search inside comments for configured annotations.
 
+    A successful match against the regex will return two groups of interest: 'token'
+    and 'data'.
+
+    This regular expression supports annotation tokens that span multiple lines. To do
+    so, prefix each line after the first by two leading spaces. E.g:
+
+        .. pii: First line
+          second line
+
+    Unfortunately, the indenting spaces will find their way to the content of the "token" group.
+
     Args:
         annotation_regexes: List of re.escaped annotation tokens to search for.
 
     Returns:
         Regex ready for searching comments for annotations.
     """
-    # pylint: disable=pointless-string-statement
-    r"""
-    This format string/regex finds our annotation token and choices / comments inside a comment:
+    annotation_regex = r"""
+    (?P<space>[\ \t]*)              # Leading empty spaces
+    (?P<token>{tokens})             # Python format string that will be replaced with a
+                                    # regex, escaped and then or-joined to make a list
+                                    # of the annotation tokens we're looking for
+                                    # Ex: (\.\.\ pii\:\:|\.\.\ pii\_types\:\:)
+    (?P<data>                       # Captured annotation data
+        (?:                         # non-capture mode
+            .                       # any non-newline character
+            |                       # or
```

Comment:
Nice comments, maybe update this to something like:
````
# or, start of new line of multi-line annotation data
```

Reply:
:ok_hand: